packages/deployer/internal/router-helper.js

```diff
@@ -9,6 +13,17 @@ function toPrivateConstantCase(name) {
   return name.replace(/(?<![A-Z])[A-Z]/g, '_$&').toUpperCase();
 }
 
+function getRouterSource() {
```

Comment:
This use to live in router-source-validator. Figured it could be used elsewhere.